legal-api/src/legal_api/services/authz.py

```diff
@@ -343,7 +343,8 @@ def get_allowable_filings_dict():
                 'alteration': {
                     'legalTypes': ['BC', 'BEN', 'ULC', 'CC'],
                     'blockerChecks': {
-                        'business': [BusinessBlocker.DEFAULT]
+                        'business': [BusinessBlocker.DEFAULT],
+                        'validStateFilings': ['limitedRestoration', 'limitedRestorationExtension']
```

Comment:
This wont work. It says a user can only file an alteration when its in limitedRestoration or limitedRestorationExtension